scripts/install-deps.sh

```diff
@@ -164,10 +186,12 @@ do_install_aduc_packages() {
     # The latest version of gcc available on Debian is gcc-6. We install that version if we are
     # building for Debian, otherwise we install gcc-8 for Ubuntu.
     OS=$(lsb_release --short --id)
-    if [[ $OS == "debian" && $VER == "9" ]]; then
+    if [[ $OS == "Debian" && $VER == "9" ]]; then
         $SUDO apt-get install --yes gcc-6 g++-6 || return
     elif [[ $OS == "Debian" && $VER == "11" ]]; then
         $SUDO apt-get install --yes gcc-10 g++-10 || return
+    elif [[ $OS == "Ubuntu" && $VER == "22.04" ]]; then
+        $SUDO apt-get install --yes gcc-10 g++-10 || return
```

Comment:
Could we change to not install particular one so that we do not have to maintain logic of mappings between distro + distro version?  
Just install the one that APT chooses?
e.g. 
```sh
$SUDO apt-get install --yes gcc g++
```

Reply:
sounds good, let me do it when adding support for debian 11 this quarter